lib/puppet/file_serving/content.rb

```diff
@@ -39,7 +39,7 @@ def content
     @content
   end
 
-  def to_raw
+  def to_binary
     File.new(full_path, "rb")
```

Comment:
Hmm, according to the corresponding spec test `spec/unit/file_serving/content_spec.rb` the expected return value is an open file and not a binary string.